src/types/chart/scorecard_chart.ts

```diff
@@ -10,6 +10,7 @@ export interface ScorecardChartDefinition {
   readonly background?: Color;
   readonly baselineColorUp: Color;
   readonly baselineColorDown: Color;
+  readonly extraData?: any;
```

Comment:
I'd add some tests to make sure the extradata is kept when duplicating the chart.  Otherwise looks fine :ok_hand: